packages/redux/src/raffles/actionTypes.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * Action type dispatched when the fetch raffle request starts.
+ */
+export const FETCH_RAFFLE_REQUEST =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_REQUEST';
+/**
+ * Action type dispatched when the fetch raffle request succeeds.
+ */
+export const FETCH_RAFFLE_SUCCESS =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_SUCCESS';
+/**
+ * Action type dispatched when the fetch raffle estimation request fails.
+ */
+export const FETCH_RAFFLE_ESTIMATION_FAILURE =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_ESTIMATION_FAILURE';
+/**
+ * Action type dispatched when the fetch raffle estimation request starts.
+ */
+export const FETCH_RAFFLE_ESTIMATION_REQUEST =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_ESTIMATION_REQUEST';
+/**
+ * Action type dispatched when the fetch raffle estimation request succeeds.
+ */
+export const FETCH_RAFFLE_ESTIMATION_SUCCESS =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_ESTIMATION_SUCCESS';
+/**
+ * Action type dispatched when the fetch raffle participation request fails.
+ */
+export const FETCH_RAFFLE_PARTICIPATION_FAILURE =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_PARTICIPATION_FAILURE';
+/**
+ * Action type dispatched when the fetch raffle participation request starts.
+ */
+export const FETCH_RAFFLE_PARTICIPATION_REQUEST =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_PARTICIPATION_REQUEST';
+/**
+ * Action type dispatched when the fetch raffle participation request succeeds.
+ */
+export const FETCH_RAFFLE_PARTICIPATION_SUCCESS =
+  '@farfetch/blackout-redux/FETCH_RAFFLE_PARTICIPATION_SUCCESS';
+/**
+ * Action type dispatched when the create raffle participation request fails.
+ */
+export const CREATE_RAFFLE_PARTICIPATION_FAILURE =
+  '@farfetch/blackout-redux/CREATE_RAFFLE_PARTICIPATION_FAILURE';
+/**
+ * Action type dispatched when the create raffle participation request starts.
+ */
+export const CREATE_RAFFLE_PARTICIPATION_REQUEST =
+  '@farfetch/blackout-redux/CREATE_RAFFLE_PARTICIPATION_REQUEST';
+/**
+ * Action type dispatched when the create raffle participation request succeeds.
+ */
+export const CREATE_RAFFLE_PARTICIPATION_SUCCESS =
+  '@farfetch/blackout-redux/CREATE_RAFFLE_PARTICIPATION_SUCCESS';
+/**
+ * Action type dispatched when the update raffle participation request fails.
+ */
+export const UPDATE_RAFFLE_PARTICIPATION_FAILURE =
+  '@farfetch/blackout-redux/UPDATE_RAFFLE_PARTICIPATION_FAILURE';
+/**
+ * Action type dispatched when the update raffle participation request starts.
+ */
+export const UPDATE_RAFFLE_PARTICIPATION_REQUEST =
+  '@farfetch/blackout-redux/UPDATE_RAFFLE_PARTICIPATION_REQUEST';
+/**
+ * Action type dispatched when the update raffle participation request succeeds.
+ */
+export const UPDATE_RAFFLE_PARTICIPATION_SUCCESS =
+  '@farfetch/blackout-redux/UPDATE_RAFFLE_PARTICIPATION_SUCCESS';
+/**
+ * Action type dispatched when the reset raffles occurs.
+ */
+export const RESET_RAFFLES_STATE =
+  '@farfetch/blackout-redux/RESET_RAFFLES_STATE';
+
+export const RAFFLES_ACTIONS = [
```

Comment:
These will get exported by the package to external consumers. Is this necessary? If you need to use it just inside your code and not by consumers, then create this value on the module you require (or if multiple modules need it, in another file that does not get exported at the `raffles/index` file.
The same for `RAFFLE_ACTIONS`, `PARTICIPATION_ACTIONS`, `PARTICIPATION_CREATIONS_ACTIONS` and `ESTIMATION_ACTIONS`.

Reply:
done, pls double check